doc_src/cmds/prompt_pwd.rst

```diff
@@ -19,6 +19,8 @@ To change the number of characters per path component, pass ``--dir-length=`` or
 
 To keep some components unshortened, pass ``--full-length-dirs=`` or set :envvar:`fish_prompt_pwd_full_dirs` to the number of components. This defaults to 1, keeping the last component.
 
+To print only up to a certain depth of parent directories, pass ``--dir-depth`` or set :envvar:`fish_prompt_pwd_dir_depth` to the desired depth. Setting it to 0 will print an unbounded number of parent directories, i.e. the full path. This defaults to 0.
```

Comment:
the name "depth" is slightly surprising because we're going inside out.
Though I don't really have a better option; maybe
`fish_prompt_pwd_parent_dir_count`

Reply:
Yeah, I don't love the naming. I had something involving `parent_dir` before, but it isn't *entirely* accurate: because with a value of 1 no parent dirs are printed.